task/pins/index.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs')
+const util = require('util')
+const path = require('path')
+
+const { parseIcon, combinePictoPin } = require('../../lib/svg_icon');
+
+const excludeList = [
+  // street furniture
+  'barrier',
+  'information',
+  'post-box',
+  'telephone',
+  'waste-basket',
+  // place types
+  'home',
+  'residential-community',
+  'town',
+];
```

Comment:
Is there a fundamental reason why pins couldn't be built for these icons ?  
Technically they may be associated to POIs, so we may need to use a pin for them too.

Reply:
Same as transports before, I removed them during my process as they seemed some kind of special case.
I can try to re-introduce them, though I'm not sure if they'll prove useful.